packages/fluentui/docs/src/prototypes/FormValidation/FormHooks.tsx

```diff
@@ -0,0 +1,62 @@
+import * as React from 'react';
+import { Form } from '@fluentui/react-northstar';
+import { useForm } from 'react-hook-form';
+
+const FormValidateHooks = () => {
+  const { register, handleSubmit, errors, setValue, watch, triggerValidation } = useForm({
+    mode: 'onBlur',
+  });
+
+  const onSubmit = data => {
+    console.log('errors: ', errors);
+    console.log('data: ', data);
+  };
+
+  const firstNameWatched = watch('firstName');
+
+  React.useEffect(() => {
+    register(
+      { name: 'city' },
+      {
+        required: { value: true, message: 'city is required' },
+      },
+    );
+  }, [register]);
+
+  const handleMultiChange = selectedOption => {
+    setValue('city', selectedOption);
+  };
+  return (
+    <Form onSubmit={handleSubmit(onSubmit)}>
+      <Form.Input
+        label="First Name"
+        id="first-name-hooks"
+        required
+        name="firstName"
+        value={firstNameWatched}
+        inputRef={register({
+          required: { value: true, message: 'first name is required' },
+          validate: val => (val.length <= 4 ? 'Too Short' : null),
+        })}
+        errorMessage={errors.firstName?.message}
+        showSuccessIndicator={!errors.firstName && firstNameWatched?.length >= 4}
+      />
+      <Form.Dropdown
```

Comment:
I would like to have an additional round of discussion for this to simplify the usage.

1. We can have `Form.Select` to mimic a `select` control (`multiple` is forced to be `false`) =>
2. In this case we can easily render a hidden `input` with a value and forward ref to it =>
3. In this case the integration will be more simple and be done like with an `Input` via `ref`

Reply:
Talked with @jurokapsiar about this one... It's very common to have to handle this when using custom select, it's so that in the libraries examples this is the example default when using some select:

https://react-hook-form.com/get-started#WorkwithUIlibrary

I think we are ok with it like this for now and we can change latter if we agree on it?!

Reply:
Agreed to solve it in the future.